tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -359,42 +360,27 @@ def read_tune_pbt(name, this):
     return config, sdc_file, fr_file
 
 
-def parse_flow_variables():
+def parse_tunable_variables():
     """
-    Parse the flow variables from source
-    - Code: Makefile `vars` target output
-
+    Parse the tunable variables from variables.yaml
     TODO: Tests.
-
-    Output:
-    - flow_variables: set of flow variables
     """
     cur_path = os.path.dirname(os.path.realpath(__file__))
-
-    # first, generate vars.tcl
-    makefile_path = os.path.join(cur_path, "../../../../flow/")
+    vars_path = os.path.join(cur_path, "../../../../flow/scripts")
```

Comment:
are you sure you want to use os.chdir() , why not use the path to the script as a reference instead?